packages/main/src/themes/TabContainer.css

```diff
@@ -103,6 +103,16 @@
 	display: none;
 }
 
+.ui5-tab-expand-button > [ui5-button] {
```

Comment:
Pressing and holding the "expand button" will turn the the arrow icon white. Can you fix this?